test/unit/actor/actorRuntime.test.ts

```diff
@@ -115,10 +114,10 @@ describe('ActorRuntime', () => {
   });
 
   it('should be able to fire a reminder', async () => {
-    const actorId = new ActorId(uuidv4());
+    new ActorId(uuidv4());
   });
 
   it('should be able to fire a timer', async () => {
-    const actorId = new ActorId(uuidv4());
+    new ActorId(uuidv4());
```

Comment:
Both these tests for `reminder` and `timer` don't seem to be doing anything, am I missing something or can this be cleaned up/marked as todo?

Reply:
The unit tests are not there yet for this, so we should mark them as todo